caddy.go

```diff
@@ -178,8 +178,17 @@ func (i *Instance) Restart(newCaddyfile Input) (*Instance, error) {
 	for _, s := range i.servers {
 		gs, srvOk := s.server.(GracefulServer)
 		ln, lnOk := s.listener.(Listener)
+		tlsLn, tlsLnOk := s.listener.(TLSlistener)
 		pc, pcOk := s.packet.(PacketConn)
 		if srvOk {
+			if tlsLnOk {
+				restartFds[gs.Address()] = restartTriple{server: gs, tlsListener: tlsLn}
+				continue
+			}
+			if tlsLnOk && pcOk {
```

Comment:
Does the prior if eclipse this?

Reply:
Of course it does :-(
(I think it's pretty self evident that it's hastily copied and pasted.... )
Will fix - my bad.